python/tk_framework_desktopserver/shotgun/api_v2.py

```diff
@@ -1956,13 +2002,13 @@ def _process_commands(self, commands, project, entities):
         # filter out things like the "Reload and Restart" command.
         filtered = list()
 
-        for command in commands:
-            if command["app_name"] is not None:
-                logger.debug("Keeping command %s -- it has an associated app.", command)
-                filtered.append(command)
+        for cmd in commands:
```

Comment:
The `command` variable was shadowing the `command` module imported at the top.

Reply:
👍